### DISCUSSION TITLE
CustomFun argument name change

### DISCUSSION THREAD

**Discussion by phleong** [QMCSoftware/QMCSoftware#312] [Q&A] [answered]
Hi there,

We have used the qmcpy.CustomFun class in some unit tests. At the time of the test development the version of the qmcpy package was 1.0. However, with the current version of the qmcpy, we noticed that one of the arguments to qmcpy.CustomFun class has been renamed -- the argument was dprime and now it is dimension_indv. We are wondering if this is just a variable name change or if this has other practical implications?

Thanks

**Comment by alegresor** [comment, marked as answer]
Hi @phleong, thank you for getting in touch. It is just a variable name change to distinguish between the dimension for individual solutions (previously `dprime`) and the newly supported dimension for combined solutions. Hope this is not too much of an inconvenience.
